yage-web/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+use std::rc::Rc;
+
+use wasm_bindgen::JsCast;
+use web_sys::{HtmlCanvasElement, WebGl2RenderingContext};
+
+use yage_core::{Context, GL};
+
+pub struct CanvasContext {
+    gl: Rc<GL>
+}
```

Comment:
I'm thinking of renaming this to `BrowserContext` - the `Canvas` in yage-core makes it a bit confusing otherwise.

Reply:
Yeah, BrowserContext sounds nice :)